src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.c

```diff
@@ -103,296 +103,295 @@ void EnSyatekiNiw_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80B11A94(EnSyatekiNiw* this, PlayState* play, s16 arg2) {
-    if (this->unk_254 == 0) {
-        if (arg2 == 0) {
-            this->unk_264 = 0.0f;
+void EnSyatekiNiw_UpdateRotateTargets(EnSyatekiNiw* this, PlayState* play, s16 animationType) {
+    if (this->peckTimer == 0) {
```

Comment:
this can probably just be `_UpdateRotations`?
its not just updating the targets anyway, it does actually change the rotations below that